lib/ansible/plugins/action/script.py

```diff
@@ -55,6 +55,10 @@ def run(self, tmp=None, task_vars=None):
                 self._remove_tmp_path(tmp)
                 return dict(skipped=True, msg=("skipped, since %s does not exist" % removes))
 
+        chdir = self._task.args.get('chdir')
+        if chdir:
+            chdir = os.path.abspath(chdir)
```

Comment:
This won't work as the os.path.abspath() is executed on the current machine.  Needs to execute on the remote machine instead.